templates/teamcity/0/docker-compose.yml

```diff
@@ -1,49 +1,60 @@
-teamcity-data:
+version: '2'
+services:
+  teamcity-server-lb:
+    image: rancher/load-balancer-service
+    links:
+      - teamcity-server
+    ports:
+      - ${lb_http_port}:8111
+
+  teamcity-data:
+    net: none
     image: busybox
-    tty: true
+    labels:
+      io.rancher.container.start_once: true
     volumes:
-        - /var/lib/teamcity
+      - /var/lib/teamcity
 
-teamcity-server:
-    image: sjoerdmulder/teamcity:latest
-    ports:
-        - ${http_port}:8111
+  teamcity-server:
+    image: sjoerdmulder/teamcity:${teamcity_version}
```

Comment:
Please, use concrete version instead of latest. Optionally, use teamcity_version as type enum with concrete versions.